Add New Featured Artist: Mediks news post (#8883)
ppy/osu-wiki: news/2023/2023-03-04-new-featured-artist-mediks.md

---
layout: post
title: "New Featured Artist: Mediks"
date: 2023-03-01 09:00:00 +0000
---

**Mediks** revives our Featured Artist library!

![](https://assets.ppy.sh/artists/346/header.jpg)

> Producer trio **Mediks** isn't your ordinary electronic music unit. Specializing in an unorthodox offshoot of drum & bass, these producers spill out tracks that reach a special niche in both the EDM and osu! communities, best known to circle clickers by their million-plays beatmap of [Outbreak](https://osu.ppy.sh/beatmapsets/527082)!

Reviewer: ```suggestion
Producer trio **Mediks** isn't your ordinary electronic music unit. Specialising in an unorthodox offshoot of drum & bass, these producers spill out tracks that reach a special niche in both the EDM and osu! communities, best known to circle clickers by their million-play beatmap of [Outbreak](https://osu.ppy.sh/beatmapsets/527082)!
```

**6** uncommon takes on drum & bass can be found on [**Mediks**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/346), including everything from their iconic *Outbreak* EP.

Sometimes we aren't permitted to promote Featured Artists alongside our usual gimmicky videos. This is one of those times.

Instead, you can preview *every track* on **Mediks**'s listing right here:

### Mediks - Outbreak

Check out [this legendary map](https://osu.ppy.sh/beatmapsets/527082) hosted by [Strategas](https://osu.ppy.sh/users/2971837)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/346/Outbreak%2FMediks%20-%20Outbreak.mp3">
</audio>

### Mediks - Nightmare

Or you can try out [this sequel map](https://osu.ppy.sh/beatmapsets/550808) by [the same mapper](https://osu.ppy.sh/users/2971837)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/346/Outbreak/Mediks%20-%20Nightmare.mp3">
</audio>

### Mediks - Doomsday

> Finish your circle clicking session with [this final map](https://osu.ppy.sh/beatmapsets/545677) hosted by [you-know-who](https://osu.ppy.sh/beatmapsets/545677).

Reviewer: ```suggestion
Finish your circle clicking session with [this final map](https://osu.ppy.sh/beatmapsets/545677) hosted by [you-know-who](https://osu.ppy.sh/users/2971837).
```

<audio controls>
    <source src="https://assets.ppy.sh/artists/346/Outbreak/Mediks%20-%20Doomsday.mp3">
</audio>

> ### Mediks - Blown Away - Raise Spirit Remix

Reviewer: ```suggestion
### Mediks feat. Astronaut - Blown Away (Raise Spirit Remix)
```

right?

![image](https://user-images.githubusercontent.com/36758269/222887529-32db2297-3f21-4f4e-92af-b4edcebeb142.png)
